router/server.go

```diff
@@ -67,9 +68,19 @@ func main() {
 	tcpIP := flag.String("tcpip", "", "tcp router listen ip")
 	tcpRangeStart := flag.Int("tcp-range-start", 3000, "tcp port range start")
 	tcpRangeEnd := flag.Int("tcp-range-end", 3500, "tcp port range end")
+	certFile := flag.String("tlscert", "", "TLS (SSL) cert file in pem format")
+	keyFile := flag.String("tlskey", "", "TLS (SSL) key file in pem format")
```

Comment:
also, `tlscert`/`tlskey` was the first name that popped into my head, lemme know if there's a better term for these.